study-builder/studies/atcp/medical-history.conf

```diff
@@ -0,0 +1,6111 @@
+                    "placeholderTemplate": null,
+                    "additionalInfoHeaderTemplate": null,
+                    "additionalInfoFooterTemplate": null,
+                    "validations": [],
+                    "suggestionType": "NONE",
+                    "suggestions": [],
+                    "promptTemplate": {
+                      "templateType": "HTML",
+                      "templateText": "$telephone_number",
+                      "variables": [
+                        {
+                          "name": "telephone_number",
+                          "translations": [
+                            {
+                              "language": "en",
+                              "text": ${i18n.en.medical_history_telephone_number}
+                            }
+                          ]
+                        }
+                      ]
+                    }
+                  }
+                ]
+                "stableId": "WHO_MADE_DIAGNOSIS"
+              },
+              "blockType": "QUESTION",
+              "shownExpr": null
+            },
+            {
+              "question": {
+                include required("../snippets/composite-question.conf"),
+                "hideNumber": true,
+                "promptTemplate": {
+                  "templateType": "HTML",
+                  "templateText": """<p>$diagnosis</p>""",
+                  "variables": [
+                    {
+                      "name": "diagnosis",
+                      "translations": [
+                        {
+                          "language": "en",
+                          "text": ${i18n.en.medical_history_where_made_diagnosis}
+                        }
+                      ]
+                    }
+                  ]
+                },
+                "children": [
+                  {
+                    "stableId": "HOSPITAL",
+                    "questionType": "TEXT",
+                    "inputType": "TEXT",
+                    "isRestricted": false,
+                    "isDeprecated": false,
+                    "hideNumber": true,
+                    "placeholderTemplate": null,
+                    "additionalInfoHeaderTemplate": null,
+                    "additionalInfoFooterTemplate": null,
+                    "validations": [],
+                    "suggestionType": "NONE",
+                    "suggestions": [],
+                    "promptTemplate": {
+                      "templateType": "HTML",
+                      "templateText": "$hospital",
+                      "variables": [
+                        {
+                          "name": "hospital",
+                          "translations": [
+                            {
+                              "language": "en",
+                              "text": ${i18n.en.medical_history_where_made_diagnosis_hospital}
+                            }
+                          ]
+                        }
```

Comment:
We also have a PHYSICIAN and an INSTITUTION component that have autocomplete for a large number of hospitals and institutions. Might make sense to use the component instead of just plain strings.
If there are some institutions that are widely used in A-T, we can ensure that they appear in the autocomplete as well. They are maintained in the back-end database.